deployment/helm/values.yaml

```diff
@@ -91,7 +98,12 @@ agent:
     tag:
     # pullPolicy is the Akri Agent pull policy
     pullPolicy: ""
-  securityContext: {}
+  # ensures container doesn't run with unnecessary priviledges
+  securityContext:
+    allowPrivilegeEscalation: false
```

Comment:
This likely will cause the agent to fail as it needs [to run as privileged](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#device-plugin-deployment) to create device plugins with the kubelet

Reply:
removed the limitation for agent

Reply:
Can we explicitly state privileged access rather than removing and it defaulting to that. This security context from the [nvidia GPU device plugin should work](https://github.com/NVIDIA/k8s-device-plugin/blob/main/nvidia-device-plugin.yml#L46)

Reply:
thanks for the suggestion and sorry for the late reply. Updated the security context of agent to follow the nvidia gpu device plugin